WebRoot/js/zimbraAdmin/cos/view/ZaCosXFormView.js

```diff
@@ -1170,11 +1178,17 @@ ZaCosXFormView.myXFormModifier = function(xFormObject, entry) {
                 items :[
                 {ref:ZaCos.A_zimbraPrefImapEnabled, type:_CHECKBOX_,
                     msgName:ZaMsg.LBL_zimbraPrefImapEnabled,
-                    label:ZaMsg.LBL_zimbraPrefImapEnabled, trueValue:"TRUE", falseValue:"FALSE"
+                    label:ZaMsg.LBL_zimbraPrefImapEnabled, trueValue:"TRUE", falseValue:"FALSE",
+                    enableDisableChecks: [ function() {
+                        return ZaCosXFormView.isAttributeDefined.call(this, ZaCos.A_zimbraPrefImapEnabled);
+                    }]
                 },
                 {ref:ZaCos.A_zimbraPrefPop3Enabled, type:_CHECKBOX_,
                     msgName:ZaMsg.LBL_zimbraPrefPop3Enabled,
-                    label:ZaMsg.LBL_zimbraPrefPop3Enabled, trueValue:"TRUE", falseValue:"FALSE"
+                    label:ZaMsg.LBL_zimbraPrefPop3Enabled, trueValue:"TRUE", falseValue:"FALSE",
+                    enableDisableChecks: [ function() {
```

Comment:
is this disabling the option or hiding it?

Reply:
this disables option, instead we should hide it

Reply:
sure will hide it